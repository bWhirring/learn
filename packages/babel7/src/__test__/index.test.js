import React from "react";
import App from "../App";
import renderer from "react-test-renderer";

test("test App.js", () => {
  const component = renderer.create(<App />);
  let tree = component.toJSON();
  // console.log(tree);
  expect(tree).toMatchSnapshot();
});
