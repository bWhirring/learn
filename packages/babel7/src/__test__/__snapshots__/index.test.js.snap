// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test App.js 1`] = `
<div>
  test
</div>
`;
